Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm

```diff
@@ -106,7 +106,20 @@ macro dispatchAfterCall(size, opcodeStruct, valueProfileName, dstVirtualRegister
     metadata(size, opcodeStruct, t2, t3)
     valueProfile(opcodeStruct, valueProfileName, t2, r1, r0)
     dispatch()
+end
 
+macro dispatchAfterTailCall(size, opcodeStruct, valueProfileName, dstVirtualRegister, dispatch)
+    loadi ArgumentCountIncludingThis + TagOffset[cfr], PC
+    if C_LOOP or C_LOOP_WIN
+        # On non C_LOOP builds, CSR restore takes care of this.
+        loadp CodeBlock[cfr], PB
+        loadp CodeBlock::m_instructionsRawPointer[PB], PB
+    end
+    get(size, opcodeStruct, dstVirtualRegister, t3)
+    storei r1, TagOffset[cfr, t3, 8]
+    storei r0, PayloadOffset[cfr, t3, 8]
+    metadata(size, opcodeStruct, t2, t3)
+    dispatch()
```

Comment:
Why is this needed? Shouldn't we not be returning to this function at this point? Could this just be a no-op/breakpoint?

Reply:
This is really silly, but we have slow path case, which looks like doing non-tail-call. So we still need this, (and we should fix it eventually).